src/mainwindow.cpp

```diff
@@ -165,19 +170,48 @@ static const IdeSettings ideSettings[] = {
 #else
     static const int ideSettingsSize = sizeof(ideSettings) / sizeof(IdeSettings);
 #endif
+
+bool isIntel()
+{
+    QList<Solid::Device> list = Solid::Device::listFromType(Solid::DeviceInterface::Processor, QString());
+    Solid::Device device = list[0];
```

Comment:
just to be pedantic - can you make sure we never crash here if the list returned is empty (shouldn't happen, but still). i.e. return false when the list is empty